src/app/content/search/components/SearchResultsSidebar/index.spec.tsx

```diff
@@ -175,4 +178,30 @@ describe('SearchResultsSidebar', () => {
 
     expect(dispatch).toHaveBeenCalledWith(clearSearch());
   });
+
+  it('search sidebar header is visible on every new search', () => {
+
+    const {tree} = renderToDom(render());
+    store.dispatch(requestSearch('cool search'));
+    store.dispatch(
+      receiveSearchResults(
+        makeSearchResults([makeSearchResultHit({ book: archiveBook, page })])
```

Comment:
if you don't care what the results are you don't have to pass anything to the `makeSearchResults` helper